### DISCUSSION TITLE
Can anyone help me run this web page containing AS3

### DISCUSSION THREAD

**Discussion by Psunabudes** [ruffle-rs/ruffle#7291] [General]
Can anyone help me runing this web page containing AS3:

https://www.bridgebase.com/client/currelease3/bbo.php?cb=IAVrsRNMqQ

**Comment by sodscg** [comment]
"  I get a message saying that "This content is not yet supported by Ruffle".

Currently Ruffle only supports games written in ActionScript 1 and 2. This includes all games before 2006 and only some games released later.

When will Actionscript 3 games work?

There is no single release date. Development is in progress, and as it progresses, simple content will start working first, while more complex content will follow later. However, as of right now (January 2022),  any kind of support will take at least several months to develop. 

If you’re wondering why ActionScript 3 support is taking a long time to develop, see https://github.com/ruffle-rs/ruffle/issues/4537#issuecomment-870982834 for an in-depth explanation.
"

**Comment by n0samu** [comment]
You can look into restoring Flash Player on your system. I have some instructions: https://gist.github.com/n0samu/06bb20c24a67df4b3e39b3aeca44a86a
